functions/Test-DbaDatabaseOwner.ps1

```diff
@@ -109,14 +109,13 @@ that TargetLogin must be a valid security principal that exists on the target se
 					Continue
 				}
 			}
-			
+			#use online/available dbs
+            $dbs = $server.Databases | Where-Object {$_.Status -eq 'Normal'}
```

Comment:
This is also filtering out databases which are not in a normal state, not sure myself whether this is desirable. As a user, I might expect to see those databases when I do -Detailed, for example?